samples/boards/nrf52/system_off/README.rst

```diff
@@ -0,0 +1,52 @@
+.. _nrf5x-system-off-sample:
+
+nRF5x System Off demo
+#####################
+
+Overview
+********
+
+This sample can be used for basic power measurement and as an example of
+deep sleep on Nordic platforms.  The functional behavior is:
+* Busy-wait for 1 second
```

Comment:
Add a blank line before the first item in a list

Reply:
done